akka-persistence/src/main/scala/akka/persistence/package.scala

```diff
@@ -12,3 +12,17 @@ package object persistence {
       else x.persistenceId.compareTo(y.persistenceId)
   }
 }
+
+package persistence {
+
+  /**
+   * INTERNAL API.
+   *
+   * Internal persistence extension messages extend this trait.
+   *
+   * Helps persistence plugin developers to differentiate
+   * internal persistence extension messages from their custom plugin messages.
+   */
+  private[persistence] trait PersistenceMessage
+
+}
```

Comment:
I would prefer that you define this trait as a top level trait in `JournalProtocol.scala`